### ISSUE TITLE
Issue conencting the robot via WiFi 

### ISSUE THREAD

**Issue by deBrian07** [Boomaa23/open-ds#20]
When I connect to the robot using WiFi, it didn't show any information from the robot. Does this DS work with robots that connect using WiFi? Or only with USB? 

I've attached a screenshot of this situation after connecting to the robot via WiFi, please take a look.

<img width="667" alt="Screenshot 2022-12-05 at 10 11 34 PM" src="https://user-images.githubusercontent.com/94733710/205835272-33c806e7-fd02-4991-8846-a7a40dfe569c.png">

**Comment by Boomaa23** [comment]
Yes! OpenDS works with robots that connect over WiFi (/ethernet). The USB was actually an addon feature. This is probably an issue with the M1 Mac (/ARM platforms in general), which I haven't been able to get working yet as I haven't had time to work on OpenDS for a while and there have been some bugs. See #18 (and to a lesser extent #19) for more details - also let me know if the JAR in that thread works for you or it's confirmed a bigger issue. I will fix it as soon as I can!

**Comment by Boomaa23** [comment]
Yeah okay same problems as the previous issue. I will try to fix it soon!

**Comment by Boomaa23** [comment]
Hey @deBrian07! I may have identified a bug while going through some other changes. Please let me know if this JAR works: 
[open-ds-v0.2.4-SNAPSHOT-jar-with-dependencies.zip](https://github.com/Boomaa23/open-ds/files/10234660/open-ds-v0.2.4-SNAPSHOT-jar-with-dependencies.zip)

**Comment by deBrian07** [comment]
Hello, I tried the JAR file you attached, but I still get the same issue. When I connect the robot using Wi-Fi, the driver station still doesn't show any robot information. The shuffleboard is still empty, so I think it's not connected to the robot.

**Comment by Boomaa23** [comment]
If you try the JAR above on the windows laptop that you confirmed to work, does that work? Also can you run it from a terminal to see the debug output:
`java -jar (the path and name of the jar here) --debug`

**Comment by deBrian07** [comment]
The older version works on Windows, but I'm trying to use it for my MacBook Pro M1. I'll try running it from terminal when I have access to the robot.

**Comment by Boomaa23** [comment]
Sounds good. Please also try the newer version (the jar in this thread) on the windows machine so I can see if it's a platform issue or internal issue.

**Comment by deBrian07** [comment]
I just tried it on my macbook with the robot, the log is showing, but the shuffle board and the enabled button still doesn't work. I tried both Wi-Fi and USB (I clicked the use USB button), but non of them works. When I connect using USB, the log says "Connecting using USB", but nothing else after. However, I do see an exception and some errors in the log. I've attached the log below. 
<img width="718" alt="image" src="https://user-images.githubusercontent.com/94733710/208211227-63256d39-4815-4bdc-a391-422545f12bf1.png">

**Comment by Boomaa23** [comment]
Please try the windows laptop as well with that same JAR (not the latest, the one from this thread)

**Comment by Boomaa23** [comment]
Actually, please try this one on both machines. I've updated the hotkey plugin so it should not throw that exception.
[open-ds-v0.2.4-SNAPSHOT-jar-with-dependencies-v2.zip](https://github.com/Boomaa23/open-ds/files/10250421/open-ds-v0.2.4-SNAPSHOT-jar-with-dependencies-v2.zip)

**Comment by deBrian07** [comment]
Got it, but I don't have access to the robot until Janurary, I'll make sure to test it on the robot once we get back. Thanks!

**Comment by Boomaa23** [comment]
Sorry one more time, please try this JAR when you get robot access: [d49c019.jar](https://aptapus.net/opends/d49c019.jar)

**Comment by Boomaa23** [comment]
Hi, just wanted to bump this. If you are no longer having this problem, please close this issue.

**Comment by deBrian07** [comment]
I'm sorry for not getting back to you sooner. I didn't have a chance to try out the JAR file you sent in the last comment. I'll try it on the robot the following Monday if I get a chance. Sorry about that.
